### DISCUSSION TITLE
How can i contribute to your repository ?

### DISCUSSION THREAD

**Discussion by namanGupta-ar** [reactplay/react-play#1430] [Q&A]
As someone new to the world of open source, I am eager to learn and contribute to meaningful projects, and your repository caught my attention. I am genuinely interested in becoming a part of your community and contributing in any way I can.

Could you please guide me on how I can get started? Whether it's addressing specific issues, contributing code, or helping with documentation, I am open to any suggestions. I understand the importance of teamwork and collaboration in open source, and I am excited about the opportunity to contribute to your project.

**Comment by priyankarpal** [comment]
hello @namanGupta-ar welcome to the community! Please check the contributing guide, README to get started https://github.com/reactplay/react-play/blob/main/README.md
